core/templates/dev/head/pages/exploration_player/progress_nav_directive.html

```diff
@@ -0,0 +1,12 @@
+<div ng-if="hasNext() || hasPrevious()" layout="row" layout-align="space-between center" style="background-color: white;">
+  <!-- Wrapping div on first button keeps the second button right-aligned
+    even when the first button is not present. -->
+  <div>
+    <md-button ng-if="hasPrevious()" ng-click="changeCard(currentCardIndex-1)" style="margin: 0px 0px 12px 12px;">
```

Comment:
Prefer 0 instead of 0px (i.e. don't need units for 0), ditto below.

Reply:
Fixed.